editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -702,15 +702,27 @@ void EditorAssetLibrary::_image_update(bool use_cache, bool final, const PoolByt
 		Ref<Image> image = Ref<Image>(memnew(Image(r.ptr(), len)));
 
 		if (!image->empty()) {
-			float max_height = 10000;
 			switch (image_queue[p_queue_id].image_type) {
-				case IMAGE_QUEUE_ICON: max_height = 80; break;
-				case IMAGE_QUEUE_THUMBNAIL: max_height = 80; break;
-				case IMAGE_QUEUE_SCREENSHOT: max_height = 345; break;
-			}
-			float scale_ratio = max_height / image->get_height();
-			if (scale_ratio < 1) {
-				image->resize(image->get_width() * scale_ratio, image->get_height() * scale_ratio, Image::INTERPOLATE_CUBIC);
+				case IMAGE_QUEUE_ICON:
+
+					image->resize(80 * EDSCALE, 80 * EDSCALE, Image::INTERPOLATE_CUBIC);
+
+					break;
+				case IMAGE_QUEUE_THUMBNAIL:
+
+					image->resize(134 * EDSCALE, 69 * EDSCALE, Image::INTERPOLATE_CUBIC);
```

Comment:
are you sure the backend always produce thumbnails with this aspect ratio, i dont see why we should asume such thing, as far as i can tell a thumbnail should also preserve its aspect ratio, with the icon it kinda makes sense but with this im not sure.